dev/reference/sql/statements/recover-table.md

```diff
@@ -21,6 +21,30 @@ RECOVER TABLE table_name
 RECOVER TABLE BY JOB ddl_job_id
 ```
 
+## **注意事项**
```

Comment:
Do you really want to apply the 2nd-level heading here? If not, please use the note format.

Reply:
yep

Reply:
Which one? 2nd-level heading or a Note?